backtrace-library/build.gradle

```diff
@@ -12,12 +12,10 @@ androidGitVersion {
 android {
 
     compileSdkVersion 33
-
+    namespace "backtraceio.library"
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 33
-        versionName androidGitVersion.name()
```

Comment:
why?

Reply:
AGP in the higher version removed it for me. I bring it back to limit the scope.